backend/pom.xml

```diff
@@ -59,6 +59,12 @@
             <artifactId>spring-boot-autoconfigure</artifactId>
             <version>2.3.4.RELEASE</version>
         </dependency>
+        <!-- Encryption & Decryption -->
+        <dependency>
+            <groupId>com.github.ulisesbocchio</groupId>
+            <artifactId>jasypt-spring-boot-starter</artifactId>
+            <version>3.0.3</version>
+        </dependency>
```

Comment:
Out of curiosity, why jasypt over spring security?

Reply:
I used jasypt because of this article https://www.geeksforgeeks.org/how-to-encrypt-passwords-in-a-spring-boot-project-using-jasypt/.